CHANGELOG.yaml

```
@@ -1,3 +1,10 @@
+3.6.6:
```

This should be `master` because the change is not released yet. Once released, this is changed to the version by @codenirvana 

```suggestion
master:
```

---

Fixed.